### PR TITLE
v2v:add TC 'Convert a guest with nonexistent iso attached'

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz#1903953

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.nonexist_iso.esx_70.rhev.rhv_upload.it_default: STARTED
█ 100% [****************************************]-]
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.nonexist_iso.esx_70.rhev.rhv_upload.it_default: PASS (1098.08 s)

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks.:)

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -211,6 +211,10 @@
                     checkpoint = 'genid_xml'
                     main_vm = VM_NAME_ESX70_GENID_WIN2019_V2V_EXAMPLE
                     os_version = "win2019"
+                - nonexist_iso:
+                    only esx_70
+                    version_requried = "[libvirt-7.0.0-1,)"
+                    main_vm = VM_NAME_ESX70_NONEXIST_ISO_WIN2022R2_V2V_EXAMPLE
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:217]: If no special requirement of Windows, It's better to use Linux guest.

Comment (vwu-vera) [reply v2v/tests/cfg/function_test_esx.cfg]: Thanks for advice. Has created the new rhel VM with nonexistence iso on 7.0 and modified the code.

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.nonexist_iso.esx_70.rhev.rhv_upload.it_default: STARTED
█ 100% [****************************************]
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.nonexist_iso.esx_70.rhev.rhv_upload.it_default: PASS (3084.39 s)

**Review by xiaodwan — APPROVED** [review]

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**